open_spiel/algorithms/is_mcts_test.cc

```diff
@@ -27,7 +27,7 @@ namespace {
 
 constexpr const int kSeed = 93879211;
 
-void PlayGame(const Game& game, algorithms::ISMCTSBot* bot, std::mt19937* rng) {
+void PlayGame(const Game &game, algorithms::ISMCTSBot *bot, std::mt19937 *rng) {
```

Comment:
@Syor Please go over the changed files and revert all unncessary whitespace changes.